dask_sql/physical/rel/custom/create_experiment.py

```diff
@@ -174,7 +166,11 @@ def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> DataContai
 
             search = ExperimentClass(model, {**parameters}, **experiment_kwargs)
             logger.info(tune_fit_kwargs)
-            search.fit(X, y, **tune_fit_kwargs)
+            search.fit(
+                X.to_dask_array(lengths=True),
```

Comment:
Could experimentClass be a gpu based model or is it limited to cpu based ones only?

Reply:
I'm not sure, since I think every example I've seen with `experiment_class` has been with a CPU `dask_ml` model... I can try to get a better idea of the scope when the pytests in https://github.com/dask-contrib/dask-sql/pull/886 are updated with other non-`dask_ml` models. We can see about adding GPU tests there too.